sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -751,6 +751,22 @@ object SQLConf {
     .checkValues(HiveCaseSensitiveInferenceMode.values.map(_.toString))
     .createWithDefault(HiveCaseSensitiveInferenceMode.NEVER_INFER.toString)
 
+  val HIVE_BLOBSTORE_SUPPORTED_SCHEMES =
+    buildConf("spark.sql.hive.blobstore.supported.schemes")
+      .doc("Comma-separated list of supported blobstore schemes.")
```

Comment:
Plz add `.version("3.1.0")`, see: https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L189

Reply:
Added the lines into the new configurations.